guides/file-upload.md

```diff
@@ -0,0 +1,42 @@
+# File Upload
+
+How about file upload? Is it possible?
+
+## Basic File Upload
+
+```ts
+import { Hono } from 'hono'
+
+const app = new Hono()
+
+app.post('/upload', async (c) => {
+  const { image } = await c.req.parseBody()
+
+  if (image instanceof File) {
+    const buffer = await image.arrayBuffer()
+  }
+
+  return c.text('Uploaded!')
+})
+```
+
+## File Upload and Save to Disk
+
+```ts
+import { Hono } from 'hono'
+
+const app = new Hono()
+
+app.post('/upload', async (c) => {
+  const { image } = await c.req.parseBody()
+
+  if (image instanceof File) {
+    const arrayBuffer = await image.arrayBuffer()
+    const buffer = Buffer.from(arrayBuffer)
+
+    fs.writeFile(path.resolve(__dirname, image.name), buffer)
```

Comment:
An import needs to be added for this line, no?

Reply:
I don't think it's necessary to add one since it's an example and let the developer who use `hono` figuring out how they need to adapt the code to their deployment environment.